debezium-e2e-benchmark/py/tpc-run-test.py

```diff
@@ -0,0 +1,223 @@
+        i = i + 1
+        print(i)
+        print(count)
+        if i >= count:
+            break
+    file.close()
+
+
+def getjdbcconnection(config, tpcconfig):
+    connectiontype = config['config']['connector.class'].split('.')[3]
+    jdbctype = 'jdbc:' + connectiontype + '://'
+    if connectiontype == 'oracle':
+        jdbctype = 'jdbc:oracle:thin:@'
+    conn = jaydebeapi.connect(tpcconfig['jdbc'][connectiontype]['jdbcdriver'], jdbctype +
+                              config['config']['database.hostname'] + ':' +
+                              config['config']['database.port'] + '/' +
+                              config['config']['database.dbname'],
+                              [config['config']['database.user'],
+                                  config['config']['database.password']],
+                              '/home/tpc/jdbcdriver/' + tpcconfig['jdbc'][connectiontype]['jar'])
+    return conn
+
+
+def main(argv):
+    with open('../register.json') as f:
+        config = json.load(f)
+    with open('tpc-config.json') as f:
+        tpcconfig = json.load(f)
+
+    print(config['config']['connector.class'])
+    print(config['name'])
+    config['name'] = 'tpc-connector'
+    print(config['name'])
+    config['config']['database.whitelist'] = 'TPC.TEST'
+
+    config['config']['database.history.kafka.topic'] = 'tpc-test'
+
+    databasetype = config['config']['connector.class']
+    print(databasetype.split('.')[3])
+    print(databasetype)
+    conn = getjdbcconnection(config, tpcconfig)
+
+    initsql(conn, config, tpcconfig)
+    createTPCTable(conn, config, tpcconfig)
+    enablecdctablesql(conn, config, tpcconfig)
+
+    print('============')
+
+    print('get status tpc connector')
+    resp = requests.get(
+        'http://' + tpcconfig['debezium.connect.server'] + '/connectors/tpc-connector/status',  verify=False)
+    print(resp.content)
+    print(resp.status_code)
+    retvalue = json.loads(resp.content)
+    print(retvalue)
+
+    print('============')
+
+    resp = requests.delete(
+        'http://' + tpcconfig['debezium.connect.server'] + '/connectors/tpc-connector',  verify=False)
+    print(resp.content)
+    print(resp.status_code)
+    # retvalue = json.loads(resp.content)
+    # print(retvalue)
+    if (resp.status_code == 404):
+        print('tpc-connector not exists')
+    else:
+        print('tpc-connector deleted')
+        pass
+
+
+    dockerbootstrapserver = config['config']['database.history.kafka.bootstrap.servers']
+    bootstrapserver = config['config']['database.history.kafka.bootstrap.servers'].split(",")
+    # check integrated test ( all in one docker)
+    if  dockerbootstrapserver == 'kafka:9092'  :
+        
+        print(bootstrapserver)
+        kafkaadmin = KafkaAdminClient(bootstrap_servers=bootstrapserver)
+
+        try:
+            kafkaadmin.delete_topics(
+                [config['config']['database.server.name'] + '.TPC.TEST'], 30)
```

Comment:
would this be the same for all databases?

Reply:
this part is running in a self-contained TPC set up for databases with Docker compose only